airbyte-integrations/connectors/source-zenhub-graphql/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,31 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "zenhub_workspace",
+        "json_schema": {},
+        "supported_sync_modes": ["full_refresh"]
+      },
+      "sync_mode": "full_refresh",
+      "destination_sync_mode": "overwrite"
+    },
+    {
+      "stream": {
+        "name": "zenhub_pipelines",
+        "json_schema": {},
+        "supported_sync_modes": ["full_refresh", "incremental"]
+      },
+      "sync_mode": "full_refresh",
+      "destination_sync_mode": "overwrite"
+    },
+    {
+      "stream": {
+        "name": "zenhub_issues",
+        "json_schema": {},
+        "supported_sync_modes": ["full_refresh", "incremental"]
```

Comment:
Do you know how you are going to do incremental syncs here?
I see the `searchIssuesByPipeline` query in the [Explorer](https://developers.zenhub.com/explorer). It looks like it has a sort order. Maybe by `udpated_at`. Is that the plan?

Reply:
Yes, I am thinking on using updated_at